src/server/pkg/deploy/assets/assets.go

```diff
@@ -692,6 +692,21 @@ func PachdDeployment(opts *AssetOpts, objectStoreBackend backend, hostPath strin
 		{Name: "EXPOSE_OBJECT_API", Value: strconv.FormatBool(opts.ExposeObjectAPI)},
 		{Name: "CLUSTER_DEPLOYMENT_ID", Value: opts.ClusterDeploymentID},
 		{Name: RequireCriticalServersOnlyEnvVar, Value: strconv.FormatBool(opts.RequireCriticalServersOnly)},
+		{
+			Name: "PACHD_POD_NAME",
+			ValueFrom: &v1.EnvVarSource{
+				FieldRef: &v1.ObjectFieldSelector{
+					APIVersion: "v1",
+					FieldPath:  "metadata.name",
+				},
+			},
+		},
+		// TODO: Setting the default explicitly to ensure the environment variable is set since we pull directly
```

Comment:
This is kind of lame, but it seems like a decent stop gap solution.

Reply:
Not sure how much it matters but this doesn't seem that lame to me, or at least I don't understand what makes it lame. This seems like the most reasonable way to get the pachd pod name. I'm a little unclear on why it is that we need that name though.

Reply:
I'm guessing this response was intended for the other comment which refers to the new `PachdPodName` field in the config. I think it is lame because we have a separate field for the pachd and worker specific config, when there should be just one field in the global config called `PodName`. Refactoring out `PPS_POD_NAME` seems a bit risky for the time being.